src/main/java/com/minecolonies/coremod/items/ModItems.java

```diff
@@ -28,10 +30,37 @@
     /**
      * Private constructor to hide the implicit public one.
      */
-    private ModItems()
+    public ModItems()
     {
         /*
          * Intentionally left empty.
          */
     }
+
+    /**
+     * Initates all the blocks. At the correct time.
+     * @param registry
+     */
+    public static void init(final IForgeRegistry<Item> registry)
+    {
+        supplyChest = new ItemSupplyChestDeployer();
+        buildTool = new ItemBuildTool();
+        scanTool = new ItemScanTool();
+        permTool = new ItemScepterPermission();
+        caliper = new ItemCaliper();
+        scepterGuard = new ItemScepterGuard();
+        supplyCamp = new ItemSupplyCampDeployer();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Synchronize this lazy initialization of 'supplyCamp' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2444)